config/samples/osbuilder_v1alpha1_osbuildenvconfig.yaml

```diff
@@ -23,5 +23,5 @@ spec:
         name: osbuild-s3-ca-bundle
       region: us-east-1
       bucket: osbuild-images
-      endpoint: "http://minio:9000"
+      endpoint: "https://minio-s3-osbuild.apps.my-cluster.example.com"
```

Comment:
Please use an environment variable for cluster-domain

Reply:
I can't do that here. There's nothing to resolve the environment variable

Reply:
Maybe it's better to use a template, I'm not really sure

Reply:
We can use `envsubst`. But that would make the apply command more complex (assuming you already set the CLUSTER_DOMAIN
`cat config/samples/osbuilder_v1alpha1_osbuildenvconfig.yaml | envsubst | oc apply -f -`

Reply:
Done